assets/pages/monthly-contributions/monthlyContributions.jsx

```diff
@@ -37,6 +37,8 @@ const store = createStore(reducer, applyMiddleware(thunkMiddleware));
 // Retrieves the contrib amount from the url and sends it to the redux store.
 store.dispatch(setContribAmount(getQueryParameter('contributionValue', '5')));
 
+store.dispatch(setPayPalButton(window.guardian.payPalButton));
```

Comment:
Yes, once we start to show the PayPal button in the page I will remove that field.